index.html

```diff
@@ -13,22 +13,30 @@
 <body>
     <!-- Toolbar -->
     <nav class="toolbar">
-        <button><img src="Assets/Icon - Menu.png" alt=""></b>
+        <button><img src="Assets/Icon - Menu.png" alt="hamburger"></button>
+        <ul>
+            <li> <a href="#portfolio">Portfolio</a> </li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
```

Comment:
You did a good job with the desktop view of your portfolio, please i'd suggest you the font-colour as shown in the figma template given so as to have a better view and allow visitors have a better experience with your portfolio 😄 

Your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/303e3bca-7a4f-4a20-8783-43dcb615e6aa)

Figma template :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/c8c06892-3d24-4754-a04c-067cd46500be)